apps/laboratory/package.json

```diff
@@ -5,7 +5,12 @@
   "scripts": {
     "dev:laboratory": "next dev",
     "build:laboratory": "next build",
-    "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
+    "build:all": "turbo run build",
```

Comment:
Turborepo and monorepo wide commands should not be ran from here. There is global `npm run build` command that does exactly this in the root, can that be reused? It also sets up clean builds properly for all packages.

You will likely need to move playwright commands and test dependencies to monorepo root as well and just target tests inside laboratory folder.